### PR TITLE
Add !

### PR DESCRIPTION
**Pull request by michaelpace**
# Reason for this change

I thought it would be a good idea to be more excited.

### CODE REVIEW

File: hello.js [hunk 1]
```diff
@@ -1 +1 @@
-console.log("Hello, world!");
\ No newline at end of file
+console.log("Hello, world!!");
```
Comment (michaelpace) [inline hello.js:1]: I think it was better before.

File: hello.js [hunk 2]
```diff
@@ -1 +1 @@
-console.log("Hello, world!");
```
Comment (Colin-Pace) [inline hello.js:1]: asdf

**Review by Colin-Pace — CHANGES_REQUESTED** [review]
Please change hello.js

**Comment by michaelpace** [comment]
@Colin-Pace — I added another ! as requested. Please give another look.

**Review by Colin-Pace — APPROVED** [review]

**Pull request merged**